docs/cpp/source/Doxyfile

```diff
@@ -70,6 +70,8 @@ RECURSIVE              = YES
 # Blacklist certain file patterns from the INPUT section.
 EXCLUDE = ../../../torch/csrc/api/include/torch/nn/pimpl-inl.h \
           ../../../torch/csrc/api/include/torch/detail
+# Increase the max node size for our large files
+DOT_GRAPH_MAX_NODES    = 100
```

Comment:
don't put this in an unrelated diff, this is what stacks are for!! Cross referencing with  https://github.com/pytorch/pytorch/pull/29041

Reply:
This is actually related. the cpp docs does not build for Exception.h without this.
See build failures on the previous commit here: https://github.com/pytorch/pytorch/runs/284960182